dashboard/urls.py

```diff
@@ -18,24 +18,27 @@
 from django.contrib.auth.decorators import login_required
 
 from django.conf import settings
-from django.conf.urls import include, url
+from django.conf.urls import include
 from django.conf.urls.static import static
 from django.urls import path, re_path
 
 from dashboard.graphql.view import DashboardGraphQLView
 
 from django.views.decorators.cache import cache_page
 
-from . import views
+from . import views, lti_new
 
 import watchman.views
-
 urlpatterns = [
     path('', views.get_home_template, name = 'home'),
     path('status/', include('watchman.urls')),
     path('status/bare_status/', watchman.views.bare_status),
 
+
     path('admin/', admin.site.urls),
+    path('login/', lti_new.login, name="login"),
```

Comment:
I know it's a little late in the review but the previous LTI had these under /lti/*.

Would it be possible to move these in there like

```
/lti/login/
/lti/launch/
/lti/jwks/

```
Having them on the top level might conflict with something now or later, as they are semi-generic paths. I *feel* like the biggest change from this will be in the documentation on the wiki.

Reply:
And these should probably only be available if LTI is enabled. You could just check the settings for that. On localhost and dev we should still support local login.